pkg/kubelet/apis/well_known_labels.go

```diff
@@ -19,80 +19,28 @@ package apis
 import (
 	"strings"
 
-	"k8s.io/apimachinery/pkg/util/sets"
-)
-
-const (
-	LabelHostname           = "kubernetes.io/hostname"
-	LabelZoneFailureDomain  = "failure-domain.beta.kubernetes.io/zone"
-	LabelMultiZoneDelimiter = "__"
-	LabelZoneRegion         = "failure-domain.beta.kubernetes.io/region"
-
-	LabelInstanceType = "beta.kubernetes.io/instance-type"
-
-	LabelOS   = "beta.kubernetes.io/os"
-	LabelArch = "beta.kubernetes.io/arch"
-
-	// GA versions of the legacy beta labels.
-	// TODO: update kubelet and controllers to set both beta and GA labels, then export these constants
-	labelZoneFailureDomainGA = "failure-domain.kubernetes.io/zone"
-	labelZoneRegionGA        = "failure-domain.kubernetes.io/region"
-	labelInstanceTypeGA      = "kubernetes.io/instance-type"
-	labelOSGA                = "kubernetes.io/os"
-	labelArchGA              = "kubernetes.io/arch"
-
-	// LabelNamespaceSuffixKubelet is an allowed label namespace suffix kubelets can self-set ([*.]kubelet.kubernetes.io/*)
-	LabelNamespaceSuffixKubelet = "kubelet.kubernetes.io"
-	// LabelNamespaceSuffixNode is an allowed label namespace suffix kubelets can self-set ([*.]node.kubernetes.io/*)
-	LabelNamespaceSuffixNode = "node.kubernetes.io"
-
-	// LabelNamespaceNodeRestriction is a forbidden label namespace that kubelets may not self-set when the NodeRestriction admission plugin is enabled
-	LabelNamespaceNodeRestriction = "node-restriction.kubernetes.io"
-)
-
-// When the --failure-domains scheduler flag is not specified,
-// DefaultFailureDomains defines the set of label keys used when TopologyKey is empty in PreferredDuringScheduling anti-affinity.
-var DefaultFailureDomains string = LabelHostname + "," + LabelZoneFailureDomain + "," + LabelZoneRegion
-
-var kubeletLabels = sets.NewString(
-	LabelHostname,
-	LabelZoneFailureDomain,
-	LabelZoneRegion,
-	LabelInstanceType,
-	LabelOS,
-	LabelArch,
-
-	labelZoneFailureDomainGA,
-	labelZoneRegionGA,
-	labelInstanceTypeGA,
-	labelOSGA,
-	labelArchGA,
-)
-
-var kubeletLabelNamespaces = sets.NewString(
```

Comment:
keep these string sets here and unexported. the only things that need to access them are kube-apiserver and kubelet, and making them public package vars lets them be mutated globally, which we don't want